redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/provision/RedbeamsProvisionFlowConfig.java

```diff
@@ -0,0 +1,65 @@
+package com.sequenceiq.redbeams.flow.redbeams.provision;
+
+import static com.sequenceiq.redbeams.flow.redbeams.provision.RedbeamsProvisionEvent.ALLOCATE_DATABASE_FAILED_EVENT;
```

Comment:
Are we allowed to do a wildcard import here? :)

Reply:
Tragically, no! It looks like static import wildcards don't work!